core/setup/kubernetes.go

```diff
@@ -1,40 +1,45 @@
 package setup
 
 import (
-//	"crypto/tls"
-//	"crypto/x509"
+//  "crypto/tls"
+//  "crypto/x509"
```

Comment:
Spaces -> tabs. I'll just rely on "go fmt" moving forward.

Leaving crypto libraries in place until I work out whether SSL support will be needed.